docs/application/native/guides/app-management/sticker.md

```diff
@@ -0,0 +1,427 @@
+# Sticker
+
+
+The Sticker feature provides users with fun experiences to share sticker information. The Sticker applications are composed of a [Sticker provider application](#provider_application) and [Sticker consumer application](#consumer_application). The Sticker provider creates stickers and developers can create their own Sticker consumer applications that use the stickers created by the Sticker provider. The Sticker provider application can store sticker details such as URI, keyword, and group name to the Sticker database. The Sticker consumer applications can retrieve sticker information using the Sticker API.
```

Comment:
Suggestion, to avoid usage of third person, suggest removing the word user. Also, since this guide is intended for developers, please change "developers" to "you". Please rephrase to:

The Sticker feature provides you with fun experiences to share sticker information. The Sticker applications are composed of a [Sticker provider application](#provider_application) and a [Sticker consumer application](#consumer_application). The Sticker provider creates stickers and you can create your own Sticker consumer applications that use the stickers created by the Sticker provider. The Sticker provider application can store sticker details such as URI, keyword, and group name to the Sticker database. The Sticker consumer applications can retrieve sticker information using the Sticker API.

Reply:
Fixed.